Pod/Classes/SwiftPhotoGallery.swift

```diff
@@ -28,9 +28,16 @@ public class SwiftPhotoGallery: UIViewController, UICollectionViewDataSource, UI
     public var imageCollectionView: UICollectionView!
     public var numberOfImages: Int = 0
 
+    public var backgroundColor: UIColor = UIColor.blackColor()
```

Comment:
This is public facing right? It should have a unit test backing it. 

Additionally, what happens if you change the color after the `imageCollectionView` is instantiated? I think it would be better to do something like this:

```swift
public var backgroundColor: UIColor {
    get {
        return imageCollectionView.backgroundColor
    }

    set(newBackgroundColor) {
        imageCollectionView.backgroundColor = newBackgroundColor
    }
}

// Then to prevent crashes when imageCollectionView hasn't been instantiated, turn it into a lazy computed property
// More Info: http://mikebuss.com/2014/06/22/lazy-initialization-swift/

public lazy var imageCollectionView: UICollectionView = self.setupCollectionView()

...

private func setupCollectionView() -> UICollectionView {
        // Set up flow layout
        flowLayout.scrollDirection = UICollectionViewScrollDirection.Horizontal
        flowLayout.minimumInteritemSpacing = 0
        flowLayout.minimumLineSpacing = 0

        // Set up collection view
        let result = UICollectionView(frame: view.bounds, collectionViewLayout: flowLayout)
        result.setTranslatesAutoresizingMaskIntoConstraints(false)
        result.registerClass(SwiftPhotoGalleryCell.self, forCellWithReuseIdentifier: "SwiftPhotoGalleryCell")
        result.dataSource = self
        result.delegate = self
        result.pagingEnabled = true

        // Set up collection view constraints
        var imageCollectionViewConstraints: [NSLayoutConstraint] = []
        imageCollectionViewConstraints.append(NSLayoutConstraint(item: imageCollectionView, attribute: .Leading, relatedBy: .Equal, toItem: view, attribute: .Leading, multiplier: 1, constant: 0))
        imageCollectionViewConstraints.append(NSLayoutConstraint(item: imageCollectionView, attribute: .Top, relatedBy: .Equal, toItem: view, attribute: .Top, multiplier: 1, constant: 0))
        imageCollectionViewConstraints.append(NSLayoutConstraint(item: imageCollectionView, attribute: .Trailing, relatedBy: .Equal, toItem: view, attribute: .Trailing, multiplier: 1, constant: 0))
        imageCollectionViewConstraints.append(NSLayoutConstraint(item: imageCollectionView, attribute: .Bottom, relatedBy: .Equal, toItem: view, attribute: .Bottom, multiplier: 1, constant: 0))

        view.addSubview(result)
        view.addConstraints(imageCollectionViewConstraints)

        // You should change this up for similar reasons. See my other comment.
        // numberOfImages = collectionView(result, numberOfItemsInSection: 0)

        result.contentSize = CGSize(width: 1000.0, height: 1.0)

        return result
    }
```